src/wp-includes/class-wp-theme.php

```diff
@@ -363,8 +363,7 @@ public function __construct( $theme_dir, $theme_root, $_child = null ) {
 			$theme_path     = $this->theme_root . '/' . $this->stylesheet;
```

Comment:
```suggestion
			$theme_path     = $this->theme_root . '/' . $this->stylesheet;
			$is_block_theme = $this->is_block_theme();
```

Instead of calling function multiple time in `if` condition set variable and use it.